packages/entity-server/src/__tests__/testUtilities/TestableEntityServer.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Tupaia MediTrak
+ * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+ */
+import * as dotenv from 'dotenv';
+
+/**
+ * Start the server
+ */
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Yeah I didn't know why this wasn't working... I will take another look, good catch!

Reply:
Okay, it's still not working and I still don't know why 😝 

I think it's got something to do with typescript since it's complaining about the types not being listed in the project dependencies... I'm gonna do a big typescript cleanup soon to get rid of all the errors across all packages so is it okay if I do it then?

Reply:
I see! Could it be that adding `@types/supertest` in the root package.json solves this?

It's a detail so happy to leave this anyway 👍

Reply:
It's there already! Very strange...